src/tooling/piral-cli/src/common/config.ts

```diff
@@ -41,6 +42,22 @@ export interface PiralCliConfig {
    * Sets the schema version to be used for pilets.
    */
   schemaVersion?: PiletSchemaVersion;
+  /**
+   * Automatically open the browser.
+   */
+  open?: boolean;
+  /**
+   * Port number.
+   */
+  port?: number;
+  /**
+   * Template language.
+   */
+  language?: SourceLanguage.ts;
```

Comment:
I think this should be `SourceLanguage` (as `SourceLanguage.ts` restricts it to only 'ts' while `SourceLanguage` is either 'js' or 'ts').

Reply:
Done.